android/src/com/mapswithme/maps/routing/RoutingErrorDialogFragment.java

```diff
@@ -80,17 +80,19 @@ private void startDownload()
       return;
     }
 
-    long size = 0;
+    long downloadSize = 0;
+    long maxMwmSize = 0;
     for (CountryItem country : mMissingMaps)
     {
       if (country.status != CountryItem.STATUS_PROGRESS &&
           country.status != CountryItem.STATUS_APPLYING)
       {
-        size += country.totalSize;
+        downloadSize += country.totalSize;
+        maxMwmSize = Math.max(maxMwmSize, country.totalSize);
```

Comment:
why additional maxMwmSize is needed? there are no updates, only new downloads here

Reply:
Hm, good point. So why GetMaxMwmSizeBytes() was there before .. Any ideas?